mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/Kit.java

```diff
@@ -1,27 +1,61 @@
 /*
- * DataLoader
- * Copyright © 2021 SolarMC Developers
  *
- * DataLoader is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
+ *  * dataloader
+ *  * Copyright © $DateInfo.year SolarMC Developers
+ *  *
+ *  * dataloader is free software: you can redistribute it and/or modify
+ *  * it under the terms of the GNU Affero General Public License as
+ *  * published by the Free Software Foundation, either version 3 of the
+ *  * License, or (at your option) any later version.
+ *  *
+ *  * dataloader is distributed in the hope that it will be useful,
+ *  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  * GNU Affero General Public License for more details.
+ *  *
+ *  * You should have received a copy of the GNU Affero General Public License
+ *  * along with dataloader. If not, see <https://www.gnu.org/licenses/>
+ *  * and navigate to version 3 of the GNU Affero General Public License.
  *
- * DataLoader is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with DataLoader. If not, see <https://www.gnu.org/licenses/>
- * and navigate to version 3 of the GNU Affero General Public License.
  */
 
 package gg.solarmc.loader.kitpvp;
 
+import java.util.Set;
+
 /**
- * Represents a kit, please fill out later
+ * Represents a kit and it's contents
+ * TODO: DO NOT LET THIS BE PACKAGE PUBLIC @
  */
-public interface Kit {
+public class Kit {
+
+    private final int id;
+    private final String name;
+    private final Set<KitPair> contents;
+
+    Kit(int id, String name, Set<KitPair> contents) {
+        this.id = id;
+        this.name = name;
+        this.contents = contents;
+    }
+
+    public int getId() {
+        return this.id;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public Set<KitPair> getContents() {
```

Comment:
How about we use a Map<Integer, KitItem> instead of a Set<KitPair>? A map of slots to items is more intuitive than a "kit pair"

(byte is considered an implementation detail)

Reply:
KitPair already contains the position of the item and the item's item data. Additionally, it is easier to go from database info to kitpair than it is to go from database info to map.

Reply:
Do you think it would at least be a good idea to name KitPair differently? Perhaps `ItemWithSlot` or `PositionedKitItem`

Reply:
fixed